rosdep/python.yaml

```diff
@@ -612,6 +612,33 @@ python-cairo:
     slackpkg:
       packages: [pycairo]
   ubuntu: [python-cairo]
+python-can:
+  alpine:
+    pip:
+      packages: [python-can]
+  arch:
+    pip:
+      packages: [python-can]
+  debian:
+    '*':
+      pip:
+        packages: [python-can]
+    buster: [python-can]
+    sid: [python-can]
```

Comment:
This will disappear once wildcard reverted, but in general we ask to not add sid rules as it's a moving target: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-rosdep-rules